staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -310,6 +330,14 @@ func (t *tracker) add(gvr schema.GroupVersionResource, obj runtime.Object, ns st
 
 	t.objects[gvr] = append(t.objects[gvr], obj)
 
+	if t.watchers[gvr] != nil && t.watchers[gvr][ns] != nil {
```

Comment:
I should be seeing more than one match to handle the all namespaces watch.  Can you add a test to demonstrate it's working?

Reply:
The "all namespace" is just an edge case of namespace that namespace itself is a `""` string, so watching/creating/updating/deleting "all namespace" resource will always manipulate the object channel stored in `t.watcher[gvr][""]`. It should work fine.

Reply:
> The "all namespace" is just an edge case of namespace that namespace itself is a "" string, so watching/creating/updating/deleting "all namespace" resource will always manipulate the object channel stored in t.watcher[gvr][""]. It should work fine like other namespaced resources .

I don't think so.  I can watch all namespaces for namespaces resources and this isn't notifying me.